docker-for-mac/edge-release-notes.md

```diff
@@ -11,6 +11,26 @@ This page contains information about Docker Desktop Edge releases. Edge releases
 For information about Stable releases, see the [Stable release notes](release-notes.md). For Docker Desktop system requirements, see
 [What to know before you install](install.md#what-to-know-before-you-install).
 
+## Docker Desktop Community 2.5.1.0
+2020-11-18
+
+> [Download](https://desktop.docker.com/mac/edge/49923/Docker.dmg)
+
+### Upgrades
+
+- [Docker Engine 20.10.0-rc1](https://github.com/docker/docker-ce/blob/master/CHANGELOG.md#20100)
+- [Compose CLI v1.0.2](https://github.com/docker/compose-cli/releases/tag/v1.0.2)
+- [Snyk v1.424.4](https://github.com/snyk/snyk/releases/tag/v1.424.4)
+- [Kubernetes 1.19.3](https://github.com/kubernetes/kubernetes/releases/tag/v1.19.3)
```

Comment:
Do we need the warning about "your cluster will be reset"?

Reply:
Good catch!

Reply:
Actually it’s a warning we put in the changelogs, I don’t see any in the release notes here in the docs…

Reply:
Ah yes right, I see it now in the 2.2.2.0 section. We missed it in the more recent ones.
I’ll add it here!